src/services/game/game.js

```diff
@@ -0,0 +1,6 @@
+export default class Game {
+    constructor() {
+        this.selectedMode = 'people';
+        this.gameModes = ['people', 'vehicles', 'spaceships'];
```

Comment:
could you please define those `people`, `vehicles`, `spaceships` variables to be reusable across project

e.g.: create variables file
```
const GAME_MODES = {
    people: 'people',
    ....
}
```
then should be `this.selectedMode = GAME_MODES.people`